samples/snippets/csharp/roslyn-sdk/SyntaxTransformationQuickStart/ConstructionCS/Program.cs

```diff
@@ -4,8 +4,8 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 // <SnippetStaticUsings>
-using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
-using static System.Console;
+using  Microsoft.CodeAnalysis.CSharp;
+using  System;
 // </SnippetStaticUsings>
```

Comment:
- You've extra space after `using`.
- The Snippet tag isn't meaningful now.
- The article that references this code will need to be updated as well to reflect the changes done in the snippet.

Reply:
Ping @BillWagner